sdk/core/core/inc/az_result.h

```diff
@@ -55,6 +55,7 @@ enum {
   AZ_ERROR_BUFFER_OVERFLOW = AZ_MAKE_ERROR(AZ_CORE_FACILITY, 2),
   AZ_ERROR_OUT_OF_MEMORY = AZ_MAKE_ERROR(AZ_CORE_FACILITY, 3),
   AZ_ERROR_NOT_IMPLEMENTED = AZ_MAKE_ERROR(AZ_CORE_FACILITY, 4),
+  AZ_ERROR_HTTP_FAILED_REQUEST = AZ_MAKE_ERROR(AZ_CORE_FACILITY, 5),
```

Comment:
Network communication deserves its own facility.

Reply:
https://github.com/Azure/azure-sdk-for-c/issues/123

Reply:
https://github.com/Azure/azure-sdk-for-c/pull/124